### PR TITLE
WIP: PaaS product page

### PR DESCRIPTION
**Pull request by elkmod**

### CODE REVIEW

File: products/paas/fastly.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Fastly
+
+
+1. Make sure you have at least Shopware 6.4.11.0
+2. Make sure `FASTLY_API_TOKEN` and `FASTLY_SERVICE_ID` are set in the environment or contact Platform.sh when its missing.
```
Comment (shyim) [inline products/paas/fastly.md:5]: maybe contact the Support to get rid of it 🤔

File: products/paas/fastly.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Fastly
+
+
+1. Make sure you have at least Shopware 6.4.11.0
```
Comment (shyim) [inline products/paas/fastly.md:4]: We should explain what the benefits are of using Fastly 🤔

File: products/paas/repository.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+# Repository
+
+The source code of your project will reside in a git-based VCS repository. You can start with a plain project, however we suggest starting with a fork of the [shopware/paas](https://github.com/shopware/paas) setup template from GitHub. You will find out more about the setup template in the [Setup Template](setup-template.md) section.
+
+## Fork and clone setup template
```
Comment (shyim) [inline products/paas/repository.md:5]: There are also integrations like for GitLab. I guess that is very important for agencies

File: products/paas/setup-template.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+This section contains configuration for environment variables or server settings. General store settings and configuration are set here. In this place you can inject custom environment variables or enable feature flags.
+
+Variables in the `env` section are automatically injected as environment variables.
+
+### hooks
+
+Lifecycle hooks are custom scripts that are called during your build and deploy processes. See more on the [deployment process](./build-deploy#push-main-branch).
+
+#### build hook
```
Comment (shyim) [inline products/paas/setup-template.md:49]: Very important to say:

- In this step you don't have access to all services, only the file system is available
- All files which are not in a `mount` are here only writeable and later steps readonly
- The best practise is to do the most stuff in the `build` as the `deploy` is blocking the web traffic

File: products/paas/setup-template.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+This script is called during the build process and builds your applications assets and disables the UI installer. You can customize this script if you need.
+
+#### deploy hook
```
Comment (shyim) [inline products/paas/setup-template.md:53]: Very important that is step is fast as possible. In the execution duration, no traffic is handled

Comment (shyim) [inline products/paas/setup-template.md:53]: Maybe we should also mention `post_deploy`. 
This hook is executed after the router is available and serves requests

File: products/paas/fastly.md (hunk 2 above)
Comment (elkmod) [reply products/paas/fastly.md:4]: what are the benefits?

**Review by shyim — APPROVED** [review]

**Pull request merged**